### PR TITLE
Rewrite poem generator

### PR DESCRIPTION
**Pull request by Ovyerus**
Completely rewrites the Lua poem generator in Python, and completes the poem command.

### CODE REVIEW

File: src/Clara/cmd/core/chatbot/chatbot.js [hunk 1]
```diff
@@ -32,17 +32,19 @@ exports.chat = {
 
 
 /**
- * creates a Response object for Dialogflow
- * @param {String} res the response to send
- * @param {String} lang the language the response is. Defaults to en
+ * Creates a response object for Dialogflow.
+ * 
+ * @param {String} res Response to send
+ * @param {String} [lang='em'] Language the response is in. Defaults to en
```
Comment (sr229) [inline src/Clara/cmd/core/chatbot/chatbot.js]: em tho blep

File: src/Clara/lib/modules/LocaleManager.js [hunk 2]
```diff
@@ -1,86 +0,0 @@
-/**
- * @file Locale manager.
```
Comment (sr229) [inline src/Clara/lib/modules/LocaleManager.js:2]: Revert this, we need the Locale manager

File: src/monika.config.js [hunk 3]
```diff
@@ -3,25 +3,26 @@
...
+            error_file: '../logs/Gateway/err.log',
+            out_file: '../logs/Gateway/out.log',
+            log_date_format: 'YYYY-MM-DD HH:mm Z',
+            watch: ['./Clara/bot.js'],
+            ignore_watch: ['data', 'logs', 'cmd', 'cache', 'node_modules'],
+            args: ['--color']
+        },
+        {
+            name: 'Monika.PoemService',
+            interpreter: '/user/bin/python3.6',
```
Comment (sr229) [inline src/monika.config.js]: isn't the interpreter ``/usr/bin/python36``?

**Review by sr229 — CHANGES_REQUESTED** [review]
blep

File: src/monika.config.js (hunk 3 above)
Comment (Ovyerus) [reply src/monika.config.js]: I dunno, it's `python3.6` on my system

Comment (sr229) [reply src/monika.config.js]: just do ``/usr/bin/python36`` tbh since its more universal

File: src/Clara/lib/modules/LocaleManager.js (hunk 2 above)
Comment (Ovyerus) [reply src/Clara/lib/modules/LocaleManager.js:2]: Then the settings system is needed to use it properly.

Comment (sr229) [reply src/Clara/lib/modules/LocaleManager.js:2]: @Ovyerus some commands still uses unless we explicitly force english only at this point

Comment (Ovyerus) [reply src/Clara/lib/modules/LocaleManager.js:2]: We can do that then.

**Review by sr229 — APPROVED** [review]
Looks good!

**Pull request merged**